api/v1alpha1/rediscacheaction_types.go

```diff
@@ -23,8 +26,10 @@ type RedisCacheActionSpec struct {
 	ActionName             RedisCacheActionName `json:"actionName"`
 	SecretName             string               `json:"secretName,omitempty"`
 	KeyVaultToStoreSecrets string               `json:"keyVaultToStoreSecrets,omitempty"`
+	ShardID                *int32               `json:"shardID,omitempty"`
 }
 
+// +kubebuilder:resource:shortName=rca,path=rediscacheactions
```

Comment:
@jpflueger  were you able to validate this on an AKS cluster?

Reply:
No, I removed it for now.